extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -28,10 +28,14 @@ attr_lists: { # amp-audio attributes that are common to both AMP and A4A format.
   name: "amp-audio-common"
   attrs: { name: "album" }
   attrs: { name: "artist" }
+  attrs: { name: "artwork" }
   attrs: { name: "controls" }
   attrs: { name: "loop"  value: "" }
   attrs: { name: "muted" value: "" }
-  attrs: { name: "poster" }
+  attrs: {
+    # deprecated use artwork instead
```

Comment:
That's correct. Remove the comment and add after `name`:

```
  deprecation: "artwork"
  deprecation_url: "https://www.ampproject.org/docs/reference/components/amp-audio"
```

See https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L104 for details and https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L1450 for example.